integration/obscurogateway/obscurogateway_test.go

```diff
@@ -45,7 +54,7 @@ const (
 )
 
 func TestObscuroGateway(t *testing.T) {
-	t.Skip("Commented it out until more testing is driven from this test")
+	// t.Skip("Commented it out until more testing is driven from this test")
```

Comment:
What if this test was removed ? I think it was mainly a placeholder, it doesnt seem to be testing anything related to the Gateway.